inlong-sort/sort-connectors/kafka/src/main/java/org/apache/inlong/sort/kafka/FlinkKafkaProducer.java

```diff
@@ -875,7 +881,11 @@ public void onCompletion(RecordMetadata metadata, Exception e) {
                         public void onCompletion(RecordMetadata metadata, Exception exception) {
                             if (exception != null && asyncException == null) {
                                 asyncException = exception;
-                                sendDirtyMetrics(rowSize, dataSize);
+                                sinkMetricData.sendDirtyMetrics(metadata.topic(), 1L, dataSize);
```

Comment:
It is inappropriate to determine whether it is dirty data here, because there may be network abnormalities, and it is recommended to be consistent with the judgment logic of dirty data archiving for dirty data.